src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1192,7 +1236,27 @@ function GlobeControls(camera, target, domElement, engine) {
     this.domElement.addEventListener('touchmove', onTouchMove.bind(this), false);
 
     // refresh control for each animation's frame
-    this.domElement.addEventListener('frameAnimation', update.bind(this), false);
+    player.addEventListener('animation-frame', update.bind(this));
+
+    function isAnimationWithoutDamping(animation) {
+        return animation && !(animation.name === 'damping-move' || animation.name === 'damping-orbit');
+    }
+
+    player.addEventListener('animation-started', (e) => {
+        if (isAnimationWithoutDamping(e.animation)) {
+            this.dispatchEvent({
+                type: 'animation-started',
+            });
+        }
+    });
+
+    player.addEventListener('animation-ended', (e) => {
```

Comment:
'`animation-started`' and '`animation-ended`' are dispatched twice when `setCameraTargetGeoPositionAdvanced(position, true)` is called